kalagraphs.module

```diff
@@ -154,10 +188,13 @@ function kalagraphs_theme() {
     }
 
     $item = [
-      'template' => $component,
-      'path' => "{$info['path']}/$component",
+      'template' => $info['template'],
+      'path' => "{$info['path']}/{$info['template']}",
       // See _kalagraphs_populate_fields() for explanation of this necessity.
-      'preprocess functions' => ['_kalagraphs_populate_fields'],
+      'preprocess functions' => [
+        '_kalagraphs_populate_fields',
+        "template_preprocess_kalagraphs_$component",
```

Comment:
I thought Josh had said the `template_blah_blah` preprocess functions get executed automatically, so I don't think you need to manually specify them here. Seems to jive with [hook_theme()](https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Render%21theme.api.php/function/hook_theme/8.3.x). We can then simply call `populate_fields()` from the template preprocess function.